llvm/lib/Target/AMDGPU/AMDGPULowerModuleLDSPass.cpp

```diff
@@ -608,6 +608,8 @@ class AMDGPULowerModuleLDS {
             ? LDSToKernelsThatNeedToAccessItIndirectly[HybridModuleRoot]
             : EmptySet;
 
+    const auto HybridModuleRootKernelsSize = HybridModuleRootKernels.size();
```

Comment:
```suggestion
    const size_t HybridModuleRootKernelsSize = HybridModuleRootKernels.size();
```

Reply:
I more meant no auto. I don't know what the type is which is exactly why you shouldn't use auto